dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -393,18 +392,6 @@ private void handleDescriptor(DescriptorLanguage descriptorType, String entry) {
      */
     protected abstract void handleStarUnstar(String entryPath, boolean unstarRequest);
 
-    /**
```

Comment:
This seems like the rare case where commenting makes more sense, it seems like an implementor would want to re-use almost all of this, even the Javadoc.